index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Portfolio</title>
+</head>
+<body>
+    <header>
+        <div id="welcome">
+            Welcome
+        </div>
+        <Nav id="menu">
+        <img src="./Images/icon-menu.png" alt="Menu Icon">
+        </Nav>
+    </header>
+<section class="bg">
+<div class="highlight">
+    Hey there. I’m Usama I’m a software developer
+</div>
+<div class="intro">
+    I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+    don’t hestiate to contact me.
+</div>
```

Comment:
- [ ] Please use `p` tag instead of `div` tag because this is a description about yourself so its a paragraph :thinking:

Reply:
@usamamaliq kindly implement the changes requested